src/eos-dropbox-app

```diff
@@ -198,10 +199,49 @@ class DropboxLauncher():
             logging.info("Not the main launcher instance. Exiting")
             self._mainloop.quit()
 
+    def _launch_dropbox(self):
+        self._launch_dropbox_daemon()
+        self._setup_config_monitor()
+
+    def _setup_config_monitor(self):
+        config = Gio.File.new_for_path(os.path.expanduser(os.path.expanduser(DROPBOX_CONFIG)))
+        self._config_monitor = config.monitor(Gio.FileMonitorFlags.NONE)
+        self._config_monitor.connect('changed', self._on_config_changed)
```

Comment:
I think you should only connect to this signal if the file has not been created yet, and disconnect as soon as it's created (and the monitor has served its purpose), so that you don't get the directory automatically opened everytime the configuration changes.